slo/rules.go

```diff
@@ -14,6 +14,8 @@ import (
 	"k8s.io/apimachinery/pkg/util/intstr"
 )
 
+const PublicLabelsPrefix = "pyrra.dev/"
```

Comment:
Do we need this one if there's `PropagationLabelsPrefix` in the same package?

Reply:
Whoops, forgot to delete this. Fixed now!